src/service/rest.js

```diff
@@ -0,0 +1,58 @@
+const get = async (path) => {
+    await wait();
+    const response = await fetch(path);
+
+    if (response.status !== 200) {
+        throw Error(await response.json());
+    }
+
+    return response.json();
+};
+
+const post = async (path, body) => {
+    const response = await fetch(path, {
+        method: 'POST',
+        headers: {
+            'content-type': 'application/json'
+        },
+        body: JSON.stringify(body)
+    });
+
+    if (response.status !== 200) {
+        throw Error(await response.json());
+    }
+
+    return response.json();
+};
+
+const authPost = async (path, body) => {
```

Comment:
Am I right that this function is not used in the project? It looks like a target for dead code elimination.